internal/controlplane/handlers_profile_test.go

```diff
@@ -15,218 +15,74 @@
 package controlplane
 
 import (
-	"errors"
-	"fmt"
 	"reflect"
 	"testing"
 
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 
-	"github.com/stacklok/minder/internal/engine"
+	"github.com/stacklok/minder/internal/profiles"
 	minderv1 "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
 )
 
-func TestValidateRuleNameAndType(t *testing.T) {
```

Comment:
I have covered these scenarios in `validator_test`. A few were duplicative and I did not copy them across.